XamlControlsGallery/ControlPages/ItemsRepeaterPage.xaml

```diff
@@ -42,6 +42,49 @@
             </Button>
         </DataTemplate>
 
+        <DataTemplate x:Key="StringDataTemplate" x:DataType="x:String">
+            <Grid Background="{StaticResource SystemControlBackgroundAccentBrush}" Margin="10">
+                <TextBlock Padding="10" Text="{x:Bind}" Foreground="{StaticResource SystemControlForegroundAltHighBrush}" HorizontalAlignment="Center" TextWrapping="Wrap" VerticalAlignment="Center"></TextBlock>
+            </Grid>
+        </DataTemplate>
+
+        <DataTemplate x:Key="StringDataTemplateColor" x:DataType="x:String">
+            <Grid Background="{StaticResource SystemControlBackgroundAccentBrush}" Margin="10, 0, 10, 10">
+                <TextBlock Padding="10" Text="{x:Bind}" Foreground="{StaticResource SystemControlForegroundAltHighBrush}" HorizontalAlignment="Center" TextWrapping="Wrap" VerticalAlignment="Center"></TextBlock>
+            </Grid>
+        </DataTemplate>
+
+        <DataTemplate x:Key="IntDataTemplate" x:DataType="x:Int32">
+            <Grid Background="{ThemeResource SystemChromeLowColor}" Margin="10">
+                <TextBlock Padding="10" Text="{x:Bind}" Foreground="{StaticResource SystemControlForegroundBaseHighBrush}" FontSize="50" HorizontalAlignment="Center" VerticalAlignment="Center"></TextBlock>
```

Comment:
This is the default color, so you don't need to specify the Foreground. Also, let's stay within existing font sizes and use HeaderTextBlockStyle. 
And finally, would you mind fixing your TextBlock declarations to end with a closing bracket instead of a hanging tag? So <TextBlock ... /> instead of <TextBlock …></TextBlock>